tools/run_tests/xds_k8s_test_driver/bin/run_test_server.py

```diff
@@ -59,6 +59,7 @@ def main(argv):
     if _SECURE.value:
         runner_kwargs.update(
             td_bootstrap_image=xds_k8s_flags.TD_BOOTSTRAP_IMAGE.value,
+            xds_server_uri=xds_flags.XDS_SERVER_URI.value,
```

Comment:
Don't we need this also in non-secure mode? ie even for baseline tests don't we want to be able to provide the staging TD URI?

Reply:
Good observation! Server doesn't use xds in the regular tests, only the client.

Reply:
Oops, yes.